src/main/java/com/devguy/devguyfx/GameController.java

```diff
@@ -168,5 +169,23 @@ public void Exit(ActionEvent actionEvent) {
     }
 
     public void about(ActionEvent actionEvent) {
+        Alert alert = new Alert(Alert.AlertType.INFORMATION);
+        alert.setTitle("DevGuy FX");
+        alert.setHeaderText(null);
+        alert.setContentText("Autoři:" + System.lineSeparator() + "Anastasiia Topor (topa00) " +
+                "Martin Čajan (cajm00)" +
+                " Ruslan Strilianyi (strr07)" +
+                " Vojtěch Zabloudil (zabv03)" +
+                " Miloš Podolský (podm10)" +
+                " Mojmír Boxan (boxm00)" +
+                " Jiří Otoupal (otoj00)"+ System.lineSeparator()+ System.lineSeparator()+
+                "Popis:" + System.lineSeparator()+
+                "Jedná se o jednoduchou Ascii hru, kde se hráč snaží utéct z firmy, sbírá předměty," +
```

Comment:
kde se snaží utéct z firmy kde programují v Javě ( bez tohohle by to nedávalo smysl jelikož tam jsou všude dialogy Jabba a střílíš C++ atd..

Reply:
Má to být v HTML dialogu, tohle neuzná